### PR TITLE
add params for oc create for OCP-25763

### PR DESCRIPTION
**Pull request by xiaocwan**
@yapei @pruan-rht @liangxia 
Please review a hot fix for oc create with params, thanks!

### CODE REVIEW

**Comment by xiaocwan** [comment]
Rebased latest fix by @liangxia and make up with the optional params

File: lib/rules/cli/4.1.yaml [hunk 1]
```diff
@@ -147,6 +147,8 @@
     :f: -f <value>
     :filename: --filename=<value>
     :record: --record=<value>
+    :reource_type: <reource_type>
```
Comment (yapei) [inline lib/rules/cli/4.1.yaml]: it should be `resource_type` and `resource_name`

Comment (xiaocwan) [reply lib/rules/cli/4.1.yaml]: @yapei thx, updated.

Comment (wewang58) [reply lib/rules/cli/4.1.yaml]: There is no resouce_type ,when "oc create --help"

File: lib/rules/cli/4.1.yaml [hunk 2]
```diff
@@ -147,6 +147,8 @@
     :f: -f <value>
     :filename: --filename=<value>
     :record: --record=<value>
+    :resource_type: <resource_type>
+    :resouce_name: <resouce_name>
```
Comment (liangxia) [inline lib/rules/cli/4.1.yaml]: 1. Still typos
2. Where are the parameters used ?

Comment (xiaocwan) [reply lib/rules/cli/4.1.yaml]: @liangxia 
1. Updated.
2. In my Open PR: https://github.com/openshift/cucushift/pull/7303/files

File: lib/rules/cli/4.1.yaml [hunk 3]
```diff
@@ -147,6 +147,8 @@
     :f: -f <value>
     :filename: --filename=<value>
     :record: --record=<value>
+    :resource_type: <resource_type>
+    :resource_name: <resource_name>
```
Comment (liangxia) [inline lib/rules/cli/4.1.yaml]: Sort the options in alpha order, so we can easily find/add/remove options.

**Comment by liangxia** [comment]
/hold
We need to squash the commit as it contains only 2 line addition, which is quite simple. 5 commits for that does not look good.
Also the commit message and description of the PR needs update, it is no longer a hot fix.

**Comment by xiaocwan** [comment]
@liangxia Is this good now?

**Comment by liangxia** [comment]
We have command "oc create user", which is added in https://github.com/openshift/verification-tests/pull/628

**Comment by xiaocwan** [comment]
> We have command "oc create user", which is added in #628

It's better to decouple the Create command, why not? Please tell me HDYT, thanks!

**Comment by liangxia** [comment]
Your command is actually using "oc create user",
$ oc create user --help
This command can be used to create a user object.

 Typically, users are created automatically during login. If automatic creation is disabled (by using the "lookup"
mapping method), users must be created manually.

 Corresponding identity and useridentitymapping objects must also be created to allow logging in as the created user.

Usage:
  oc create user USERNAME [flags]

Examples:
  # Create a user with the username "ajones" and the display name "Adam Jones"
  oc create user ajones --full-name="Adam Jones"

**Comment by xiaocwan** [comment]
Will Update OCP-25763 from feature step after the PR https://github.com/openshift/verification-tests/pull/628 merge. This PR could be closed after that rule merge.

**Pull request closed without merging**